shared/src/helpers/date/sort-dates/sort-dates.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type sortingType = 'desc' | 'asc';
+
+interface ShouldHaveCreatedAtProp {
+  createdAt: string;
+}
+
+const sortDates = <T extends ShouldHaveCreatedAtProp>(
+  items: T[],
```

Comment:
Pass there only dates
```suggestion
  dates: Date[],
```